src/game/common/bitflags.cpp

```diff
@@ -65,24 +65,6 @@ const char *BitFlags<OBJECT_STATUS_COUNT>::s_bitNamesList[] = {
     nullptr,
 };
 
-template<>
-const char *BitFlags<DISABLED_TYPE_COUNT>::s_bitNamesList[] = {
-    "DEFAULT",
```

Comment:
Why is this being moved. All the other instances are in this file

Reply:
tomsons26 refered to the existence of this file "DisabledTypes" from the original. So I moved all related things there. Note that some other BitFlag types, such as KindOf, also have their own file. I think that makes sense. They should not be cramped all into BitFlags file. Is not tidy.